src/solvers/smt2_incremental/smt_bit_vector_theory.cpp

```diff
@@ -590,8 +599,7 @@ smt_sortt smt_bit_vector_theoryt::negatet::return_sort(const smt_termt &operand)
 
 void smt_bit_vector_theoryt::negatet::validate(const smt_termt &operand)
 {
-  const auto operand_sort = operand.get_sort().cast<smt_bit_vector_sortt>();
-  INVARIANT(operand_sort, "The operand is expected to have a bit-vector sort.");
+  validate_bit_vector_sort("The", operand);
```

Comment:
Sorry, a random string literal here is not terribly informative - I found it confusing when looking at just the callsite.

Would it be possible to refactor this function so that it either uses a default argument, or is overloaded, or otherwise defaults to a "The" somehow? Using a global const for that might also be okay.

Reply:
Ok. I have added an extra overload. Default arguments only work for parameters at the right of the list. I could have used an `optionalt<std::string>` argument but `({}, operand)` would have been almost as awkward to read as the existing `("The", operand)`